ideascube/forms.py

```diff
@@ -70,14 +70,55 @@ def save(self, *args, **kwargs):
 class UserImportForm(forms.Form):
     source = forms.FileField(required=True)
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        formats = getattr(settings, 'USER_IMPORT_FORMATS', None)
+
+        if formats is None:
+            self.fields['format'] = forms.CharField(
+                required=True, initial='ideascube', widget=forms.HiddenInput)
+
+        else:
+            self.fields['format'] = forms.ChoiceField(
+                required=True, initial=formats[0][0], choices=formats)
```

Comment:
Maybe we could check here that we know how to handle the proposed format (check that `'_get_%s_mapping' % format` exists).

This would avoid us to show a option we cannot handle to the user (and so, raise a `ValidationError` the user can't fix).